firestore/FirestoreSwiftUIExample/ViewModels/RestaurantListViewModel.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  RestaurantListViewModel.swift
```

Comment:
Use copyright headers

Reply:
Ah, thanks, this slipped my mind. I'll copy over from the UIKit version.